docs/website/data/integrations.yaml

```diff
@@ -1072,7 +1072,23 @@ integrations:
     tutorials:
       - https://hub.armo.cloud
 
-
+  i2scim:
+    title: i2scim.io OPA Enabled User/Group Provisioning API
+    description: |
+      i2scim.io is an open source, Apache 2 Licensed implementation of RFC7643/7644 for cloud-native platforms. i2scim supports 
```

Comment:
😅 I happen to have come across SCIM in a previous occasion, but I'm not sure if it's widespread enough to be understood by RFC number alone... Could we please add something more here? Expand "SCIM", maybe?